taui/src/components/neighborhood-details.js

```diff
@@ -76,7 +76,7 @@ class NeighborhoodDetails extends PureComponent<Props> {
           {t('Units.Mins')}&nbsp;
           <ModesList segments={bestJourney} />&nbsp;
           {t('NeighborhoodDetails.FromOrigin')}&nbsp;
-          {currentDestination && t('TripPurpose.' + currentDestination.purpose).toLowerCase()}
+          {currentDestination && t(currentDestination.purpose).toLowerCase()}
```

Comment:
If `TripPurpose` is no longer part of `destination.purpose` then this change should no longer be necessary.